python/lsst/ts/MTAlignment/alignmentModel.py

```diff
@@ -0,0 +1,507 @@
+
+        msg = "!SET_2FACE_TOL:" + tolerance
+        data = await self.send_msg(msg)
+        return data
+
+    async def set_drift_tolerance(self, rms_tol, max_tol):
+        """
+        rms_tol default 0.050 mm
+        max_tol default 0.1 mm
+        """
+
+        msg = f"!SET_DRIFT_TOL:{rms_tol};{max_tol}"
+        data = await self.send_msg(msg)
+        return data
+
+    async def set_ls_tolerance(self, rms_tol, max_tol):
+        """
+        rms_tol default 0.050 mm
+        max_tol default 0.1 mm
+        not actually sure how this one differs from the previous...
+        """
+
+        msg = f"!SET_LS_TOL:{rms_tol};{max_tol}"
+        data = await self.send_msg(msg)
+        return data
+
+    async def load_template_file(self, filepath):
+        """
+        Load template file
+
+        filepath: String
+        """
+
+        msg = "!LOAD_SA_TEMPLATE_FILE:" + filepath
+        data = await self.send_msg(msg)
+        return data
+
+    async def set_reference_group(self, pointgroup):
+        """
+        Nominal pt grp to locate station to and provide data relative to.
+
+        pointgroup: String
+
+        """
+
+        msg = "!SET_REFERENCE_GROUP:" + pointgroup
+        data = await self.send_msg(msg)
+        return data
+
+    async def set_working_frame(self, workingframe):
+        """
+        Make workingframe the working frame:
+
+        workingframe: string
+        """
+
+        msg = "!SET_WORKING_FRAME:" + workingframe
+        data = await self.send_msg(msg)
+        return data
+
+    async def new_station(self):
+        """
+        A new station is added and made to be the active instrument.
+        """
+
+        msg = "!NEW_STATION"
+        data = await self.send_msg(msg)
+        return data
+
+    async def save_sa_jobfile(self, filepath):
+        """
+        Save a jobfile
+
+        filepath: String
+        """
+
+        msg = "!SAVE_SA_JOBFILE:" + filepath
+        data = await self.send_msg(msg)
+        return data
+
+    async def set_station_lock(self, val):
+        """
+        Prevents SA from automatically jumping stations when it detects that the tracker has drifted.
+
+        val: Boolean
+        """
+        if val:
+            msg = "!SET_STATION_LOCK:1"
+        else:
+            msg = "!SET_STATION_LOCK:0"
+        data = await self.send_msg(msg)
+        return data
+
+    async def reset_t2sa(self):
+        """
+        Reboots the T2SA and SA components
+        """
+
+        msg = "!RESET_T2SA"
+        data = await self.send_msg(msg)
+        return data
+
+    async def halt(self):
+        """
+        Commands T2SA to halt any measurement plans currently executing,
+        and return to ready state
+        """
+
+        msg = "!HALT"
+        data = await self.send_msg(msg)
+        return data
+
+    async def publish_telescope_position(self, telalt, telaz, camrot):
+        """
+        Command in which TCS informs T2SA of the telescope’s current
```

Comment:
I would suggest renaming this `set_telescope_position`. Also, it is a general way of telling T2SA the position of the telescope, not necessarily TCS... in fact the CSC is the one that will use it.